### ISSUE TITLE
level-type flat

### ISSUE THREAD

**Issue by GlennGeenen** [GlowstoneMC/Glowstone-Legacy#443]
Is level-type flat supported? It says so in the wiki but I'm unable to generate a flat world.

**Comment by ZephireNZ** [comment]
Glowstone is still very much in progress. I wouldn't be surprised if there isn't a flat world generator just yet.

**Comment by dequis** [comment]
> The world type to use for the main world, one of DEFAULT, FLAT, DEFAULT_1_1, LARGEBIOMES, or AMPLIFIED.

Nope. That's all vanilla worldgen stuff. That part of the page might be copied from the minecraft wiki.

**Comment by SpaceManiac** [comment]
That configuration option does exist, and could stand to be implemented as part of #76.
